samples/maps/demo/geoheatmap-earth-statistics/demo.details

```diff
@@ -0,0 +1,15 @@
+---
+name: Earth Statictics August 2022 - GeoHeatmap interpolation
```

Comment:
The name should be shortened - this is what is used for the titles on the [demo page](https://www.highcharts.com/demo/maps#highcharts-maps-demo-general).